docs/guides/choose-execution-mode.mdx

```diff
@@ -0,0 +1,76 @@
+   - The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.
+   - There is usually minimal delay between jobs, which can help avoid drift.
+   - If you partition your workload into multiple jobs and run them in batch mode, you can get results from individual jobs, which makes them more flexible to work with. For example, if a job's results don't meet your expectations, you can cancel the remaining jobs. Also, if one job fails, you can re-submit it instead of re-running the entire workload.
+   - Is generally less expensive than sessions.
+- **Session**
+   - All the benefits from batch mode.
+   - Dedicated and exclusive access to the QPU during the session active window.
+   - Useful for workloads that don’t have all inputs ready at the outset, for iterative workloads that require classical post-processing before the next one can run, and for experiments that need to run as tightly together as possible.
+- **Job**
+   - Easiest to use when running a small experiment.
+   - Might run sooner than batch mode.
+
+<span id="best-practices"></span>
+## Recommendations and best practices
+
+Generally, use batch mode unless you have workloads that don’t have all inputs ready at the outset.
+
+- Use **batch** mode to submit multiple primitive jobs simultaneously to shorten processing time.
+- Use **session** mode for iterative workloads, or if you need dedicated access to the QPU.
+- Always use **job** mode to submit a single primitive request.
+- Because _sessions are generally more expensive_, it is recommended that you use batch whenever you don't need the additional benefits from using sessions.
+
+To ensure the most efficient use of the execution modes, the following practices are recommended:
+
+- There is a fixed overhead associated with running a job. In general, if each of your jobs uses less than one minute of _QPU time_, consider combining several into one larger job (this applies to all execution modes). "QPU time" refers to time spent by the QPU complex to process your job.
+
+- If each of your jobs consumes more than one minute of QPU time, or if combining jobs is not practical, you can still run multiple jobs in parallel. Every job goes through both classical and quantum processing. While a QPU can process only one job at a time, up to five classical jobs can be processed in parallel. You can take advantage of this by submitting multiple jobs in [batch](/guides/run-jobs-batch#partition) or [session](/guides/run-jobs-session#two-vqe) execution mode.
+
+The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, if you are using a session to get exclusive access to a backend, consider breaking up large jobs into smaller ones and running them in parallel. This might be more cost-effective because it can reduce wall clock time.
```

Comment:
```suggestion
The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, if you are using a session to get exclusive access to a backend, consider breaking up large jobs into smaller ones and running them in parallel. This might be more cost-effective because it can reduce wall-clock time.
```